src/Page.ts

```diff
@@ -738,11 +738,16 @@ export class Page extends EventEmitter {
 
   _onDialog(event: Protocol.Page.javascriptDialogOpeningPayload): void {
     let dialogType = null;
-    if (event.type === 'alert') dialogType = Dialog.Type.Alert;
```

Comment:
this `Dialog.Type` only existed for this method and was never documented. So I removed it to save having it (and therefore having to document it).